python_modules/libraries/dagster-dbt/tox.ini

```diff
@@ -32,8 +32,8 @@ allowlist_externals =
   uv
 commands =
   !windows: /bin/bash -c '! pip list --exclude-editable | grep -e dagster'
-  cloud: pytest --numprocesses 6 --durations 10  --reruns 3 -c ../../../pyproject.toml -m "cloud" -vv {posargs}
-  core-main: pytest --numprocesses 6 --durations 10  --reruns 3 -c ../../../pyproject.toml -m "core and not snowflake and not bigquery and not derived_metadata" -vv {posargs}
-  core-derived-metadata: pytest --numprocesses 6 --durations 10 --reruns 3 -c ../../../pyproject.toml -m "derived_metadata and not snowflake and not bigquery" -vv {posargs}
+  cloud: pytest --durations 10  --reruns 3 -c ../../../pyproject.toml -m "cloud" -vv {posargs}
+  core-main: pytest --durations 10  --reruns 3 -c ../../../pyproject.toml -m "core and not snowflake and not bigquery and not derived_metadata" -vv {posargs}
+  core-derived-metadata: pytest --durations 10 --reruns 3 -c ../../../pyproject.toml -m "derived_metadata and not snowflake and not bigquery" -vv {posargs}
```

Comment:
are these just run single core if no argument is passed? I thought you were going to drop N from 6 to 3 or 4

Reply:
will be curious to see the runtimes from this BK run though

Reply:
We saved 3-4 minutes per test suite by getting rid completely of `--numprocesses`

Previous build (one test suite failed because of the issue mentioned in the description of this PR) https://buildkite.com/dagster/dagster-dagster/builds/110275#_

Current build https://buildkite.com/dagster/dagster-dagster/builds/110339#_

Reply:
And all dbt test suites are successful